lib/rules/sotd/process-document.js

```diff
@@ -25,7 +39,11 @@ exports.check = function (sr, done) {
                 }
             }
             else if (deprecatedBoilerplate === sr.norm($p.text()) || deprecatedUri === $p.find('a').attr('href')) {
-                sr.error(self, 'deprecated');
+                if (deprecatedAllowed) {
+                    sr.warning(self, 'deprecated');
+                    found = true;
+                }
+                else sr.error(self, 'deprecated');
```

Comment:
Do you think it'd be helpful to pass along the detected process as a parameter, to be displayed as part of the message?